native/cocos/audio/apple/AudioDecoder.h

```diff
@@ -105,6 +105,8 @@ class AudioDecoder {
      * @note Currently we only support 1 or 2 channels.
      */
     uint32_t getChannelCount() const;
+    /** Gets bytes per channel in one frame.*/
+    uint32_t getBytesPerChannel() const;
```

Comment:
`getBytesPerChannel` seems to be `get all bytes in a channel`. But i think that 's not what you mean.

Reply:
removed, it can be calculated after